src/ScreenGameplay.cpp

```diff
@@ -91,8 +91,7 @@ AutoScreenMessage( SM_BattleTrickLevel2 );
 AutoScreenMessage( SM_BattleTrickLevel3 );
 
 static Preference<bool> g_bCenter1Player( "Center1Player", false );
-static Preference<bool> g_bShowLyrics( "ShowLyrics", true );
-static Preference<float> g_fNetStartOffset( "NetworkStartOffset", -3.0 );
+static Preference<bool> g_bShowLyrics( "ShowLyrics", false );

```

Comment:
I don't mind this being on as well. It's kind a fun to have the if the chart includes it. I think people can manually turn it off if they're bothered.

Reply:
changed!